static/src/javascripts-legacy/projects/common/modules/experiments/ab.js

```diff
@@ -87,9 +87,12 @@ define([
     }
 
     function getActiveTests() {
-        var now = new Date();
+        var today = new Date();
+        // new Date(test.expiry) sets the expiry time to 00:00:00
+        // Using SetHours allows a test to run until the END of the expiry day
+        today.setHours(0,0,0,0);
```

Comment:
Is this setting the time forward or backward? It sets the time to the _start_ of today, ie pushes the time earlier, right?